samples/excel/85-preview-apis/data-types-formatted-number.yaml

```
@@ -0,0 +1,149 @@
+order: 1
+id: excel-data-types-formatted-number
+name: 'Data types: Formatted numbers'
+description: This sample shows how to set and get data types using the formatted number properties.
+host: EXCEL
+api_set:
+    ExcelApi: BETA (PREVIEW ONLY)
+script:
+    content: |
+        $("#set-formatted-number-date").click(() => tryCatch(setFormattedNumberDate)); 
+        $("#set-formatted-number-currency").click(() => tryCatch(setFormattedNumberCurrency)); 
+        $("#get-formatted-number").click(() => tryCatch(getFormattedNumber));
+
+        async function setFormattedNumberDate() {
+          await Excel.run(async (context) => {
+            // This method creates a formatted number data type,
+            // and sets the format of this data type as a date.
+
+            // Create a worksheet called "Sample" and activate it.
+            context.workbook.worksheets.getItemOrNullObject("Sample").delete();
+            const sheet = context.workbook.worksheets.add("Sample");
+            sheet.activate();
+            
+            // Get cell A1 and write a number formatted as a date to cell A1.
+            const dateRange = sheet.getRange("A1");
+            dateRange.valuesAsJson = [
+              [
+                {
+                  type: Excel.CellValueType.formattedNumber,
+                  basicValue: 32889.0,
+                  numberFormat: "m/d/yyyy"
+                }
+              ]
+            ];
+            await context.sync();
+          });
+        }
+
+        async function setFormattedNumberCurrency() {
+          await Excel.run(async (context) => {
+            // This method creates a formatted number data type,
+            // and sets the format of this data type as a currency.
+
+            // Get the active worksheet and a range on that sheet.
+            const sheet = context.workbook.worksheets.getActiveWorksheet();
+            const currencyRange = sheet.getRange("A2");
+
+            // Write a number formatted as currency to cell A2.
+            currencyRange.valuesAsJson = [
+              [
+                {
+                  type: Excel.CellValueType.formattedNumber,
+                  basicValue: 12.34,
+                  numberFormat: "$* #,##0.00"
+                }
+              ]
+            ];
+
+            await context.sync();
+          });
+        }
+
+        async function getFormattedNumber() {
+          await Excel.run(async (context) => {
+            // This method prints information about data types 
+            // in cells A1 and A2 to the console.
+
+            // Get the active worksheet and two ranges on that sheet.
+            const sheet = context.workbook.worksheets.getActiveWorksheet();
+            const currencyRange = sheet.getRange("A2");
+            const dateRange = sheet.getRange("A1");
+
+            // Load the property of the ranges that contains data types.
```

This sentence reads a little awkwardly because on first pass it sounds like there's a grammar error in "ranges that contains data types". Consider reworking it to avoid potential confusion.
```suggestion
            // Load the data type property of the ranges.
```